File: app/src/main/java/com/example/testlunu/net/acu/weather/Temperature.java
package com.example.testlunu.net.acu.weather;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Temperature {
    public Minimum getMinimum() {
        return minimum;
    }

    public Maximum getMaximum() {
        return maximum;
    }

    @SerializedName("Minimum")
    @Expose
    public Minimum minimum;
    @SerializedName("Maximum")
    @Expose
    public Maximum maximum;

}

File: app/src/main/java/com/example/testlunu/net/acu/weather/Temperature.kt
package com.example.testlunu.net.acu.weather

import com.example.testlunu.net.acu.weather.decompiled.Maximum
import com.example.testlunu.net.acu.weather.decompiled.Minimum
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Temperature {

    @SerializedName("Minimum")
    @Expose
    var minimum: Minimum? = null
    @SerializedName("Maximum")
    @Expose
    var maximum: Maximum? = null
}